README.md

```diff
@@ -19,8 +19,8 @@ After that, a telnet server will be up and running on the router. You can connec
 telnet <router_ip_address>
 ```
 
-* User: root
-* Password: none (just hit enter)
+* User: `root`
+* Password: none (just hit enter) or `root`
```

Comment:
Thanks! Password was changed recently in the last release, but I forgot to update it here. Could you update this line to be:
```
* Password: `root`
```